libkbfs/folder_branch_ops.go

```diff
@@ -279,6 +279,11 @@ type folderBranchOps struct {
 	// rekey with a paper key prompt, if enough time has passed.
 	// Protected by mdWriterLock
 	rekeyWithPromptTimer *time.Timer
+
+	// latestRemoteMergedRevision tracks the latest know merged revision of on
+	// server
+	muLatestRemoteMergedRevision sync.RWMutex
```

Comment:
I'd rather you rely on the `headLock` mutex instead of introducing a new one.  See below for more details.